public/scripts/browse.js

```diff
@@ -40,7 +40,24 @@
 
```

Comment:
I think this file in general could be renamed to map.js or something, since it's being used in both the browse and report views now